src/PayloadInterface.php

```diff
@@ -6,25 +6,85 @@
 {
     /**
      * @var integer All good.
+     * @deprecated 1.1.0
+     * @see \Equip\Adr\PayloadInterface::STATUS_OK
      */
     const OK = 10;
 
     /**
      * @var integer Error during domain processing.
+     * @deprecated 1.1.0
      */
     const ERROR = 20;
 
     /**
      * @var integer Invalid domain input.
+     * @deprecated 1.1.0
      */
     const INVALID = 30;
 
     /**
      * @var integer Unknown domain state.
+     * @deprecated 1.1.0
      */
     const UNKNOWN = 40;
 
     /**
+     * @var integer
+     */
+    const STATUS_OK = 'OK';
+
+    /**
+     * @var integer
+     */
+    const STATUS_CREATED = 'Created';
+
+    /**
+     * @var integer
+     */
+    const STATUS_ACCEPTED = 'Accepted';
+
+    /**
+     * @var integer
+     */
+    const STATUS_NO_CONTENT = 'No Content';
+
+    /**
+     * @var integer
+     */
+    const STATUS_MOVED_PERMANENTLY = 'Moved Permanently';
+
+    /**
+     * @var integer
+     */
+    const STATUS_FOUND = 'Found';
+
+    /**
+     * @var integer
+     */
+    const STATUS_NOT_MODIFIED = 'Not Modified';
+
+    /**
+     * @var integer
+     */
+    const STATUS_BAD_REQUEST = 'Bad Request';
+
+    /**
+     * @var integer
+     */
+    const STATUS_UNAUTHORIZED = 'Unauthorized';
+
+    /**
+     * @var integer
+     */
+    const STATUS_FORBIDDEN = 'Forbidden';
+
+    /**
+     * @var integer
+     */
+    const STATUS_NOT_FOUND = 'Not Found';
```

Comment:
Despite what I said before, we should add all of the statuses that are found in http-status package. That's safest and will reduce the need to version up later on.